### ISSUE TITLE
Loader, spin animation time

### ISSUE THREAD

**Issue by Melmoth-the-Wanderer** [Semantic-Org/Semantic-UI#6138]
Is there any way to configure spinning animation time? 
.6s is quite aggressive to me. I'd like to set it to .9-1s. I can mess with the DOM and styles, but would be nice to introduce setting for this for the sake of performance. I read the docs and couldn't find any.

**Comment by hammy2899** [comment]
There is a variable in `globals/site.variables` called `@loaderSpeed` which you can change (https://github.com/Semantic-Org/Semantic-UI/blob/master/src/themes/default/globals/site.variables#L183)

**Comment by Melmoth-the-Wanderer** [comment]
@hammy2899 thanks for the hint! Switching to buildtools would do the trick and someday I'll probably do it, when it makes more sense in case of my project.

Currently I'm using standalone file downloaded from the repo without bulid tools and was hoping to have the setting exposed in runtime, if it's worth adding extra javascript to the element.

**Comment by hammy2899** [comment]
I don't think this will be added since the `loader` is a element not a module therefore no JavaScript.

**Comment by GammaGames** [comment]
Couldn't there just be a couple modifier classes like slow, slower, or fast?

**Comment by hammy2899** [comment]
@GammaGames that could probably work!

**Comment by lubber-de** [comment]
Fixed in https://github.com/fomantic/Fomantic-UI 2.4.4

**Comment by aggregate1166877** [comment]
> Is there any way to configure spinning animation time?

If you don't mind overriding the default CSS, you can add something like this to a CSS file included after Semantic's:

```CSS
.ui.loader:after {
  animation: loader 5s linear infinite;
}
```

**Comment by GammaGames** [comment]
You can also install the community fork, which has this feature as well as a ton of bugfixes and extras: https://fomantic-ui.com/elements/loader.html#speed
